cloud/amazon/ec2_elb_lb.py

```diff
@@ -182,6 +187,10 @@
         interval: 30 # seconds
         unhealthy_threshold: 2
         healthy_threshold: 10
+    access_logs:
+        interval: 5 # minutes
```

Comment:
Since we set a default explicitly below, it might make sense to call that out in the comment e.g.

```
interval: 5 # minutes (defaults to 60)
```